db/seeds.rb

```diff
@@ -1,6 +1,6 @@
 
 item_contents = ([{ name: "Chocolate Toast", description: "Dark chocolate on toast",
-        price: "3.00", retired?: true}, { name: "Cake Toast", description: "Lemon poppyseed cake on toast", price: "4.00"},
+        price: "3.00", retired: true}, { name: "Cake Toast", description: "Lemon poppyseed cake on toast", price: "4.00"},
```

Comment:
Line is too long. [122/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.